lib/app/features/home/home_tabs/upcoming_tab/cubit/upcoming_tab_cubit.dart

```diff
@@ -14,18 +14,22 @@ class UpcomingTabCubit extends Cubit<UpcomingTabState> {
   final HomeScreenRepository homeScreenRepository;
 
   Future<void> fetchUpcomingMatches(DateTime date) async {
-    String formattedDate = DateFormat('yyyy-MM-dd').format(date);
-    emit(const LoadingMatchesState());
-    if (date.isAtSameMomentAs(DateTime.now()) || date.isAfter(DateTime.now())) {
-      try {
+    String formattedStringDate = DateFormat('yyyy-MM-dd').format(date);
```

Comment:
I think the same will work here as well. Try it

Reply:
answered above